### PR TITLE
Added page for live support account recovery

### PR DESCRIPTION
**Pull request by CoburnFuse**
Obviously a first time for me to add this.
Given how much '!hacked' is used, and how many accounts get hacked. This feels like its a good thing to add.
Thanks to Shakey for the main image that I used for instructions.

### CODE REVIEW

File: wiki/hacked-accounts.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Getting Live Support for Hacked Accounts
+
+This article covers how to get live chat support in case your account gets hacked.
+
+::: tip Note
+We can not help recovering hacked accounts in the Discord server, as we do not have access to account information.
+:::
```
Comment (itsWindows11) [inline wiki/hacked-accounts.md:7]:
```suggestion
::: tip Note
We can not help you recover hacked accounts in the Discord server, as we do not have access to account information.
:::
```
I prefer it this way, if not a typo.

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
LGTM, just a minor change's needed.

File: wiki/hacked-accounts.md (hunk 1 above)
Comment (CoburnFuse) [reply wiki/hacked-accounts.md:7]: While not a typo, it does look better. Agreed.

Comment (itsWindows11) [reply wiki/hacked-accounts.md]: Just looked it up, it appears that the verb "help" doesn't take a gerund (v. + ing) after it directly, but what's done in this suggestion is a way to write it correctly at least 🤷‍♂️

**Review by Erisa — APPROVED** [review]
lgtm, don't merge for now, will review alongside #155 a bit later today

**Review by Erisa — APPROVED** [review]

**Pull request merged**